sunpy/instr/tests/test_aia.py

```diff
@@ -21,7 +22,8 @@ def original(request):
 
 @pytest.fixture
 def prep_map(original):
-    return aiaprep(original)
+    with pytest.warns(SunpyDeprecationWarning):
```

Comment:
What does this do in the fixture? Just catches the warning so pytest doesn't raise an error for this deprecation warning?

Reply:
yep

Reply:
And also fails the test if the code *doesn't* raise a warning.

Reply:
Ah ok. So works the same in a fixture as in a normal test 👍